xapian-applications/omega/templates/query

```diff
@@ -1,4 +1,5 @@
-$httpheader{Content-Type,text/html; charset=utf-8}<html>
+$httpheader{Content-Type,text/html; charset=utf-8}
+<html>
```

Comment:
There's a reason why there wasn't a newline before `<html>` here before.

`$httpheader{}` sets a header to send on the HTTP response and expands to nothing in the output, so your updated template will result in a blank line before the `<html>` in the output, which is a little undesirable, if only cosmetically.